src/components/MenuItem.js

```diff
@@ -87,6 +88,7 @@ const MenuItem = props => (
                             <Avatar
                                 imageStyles={[styles.avatarNormal, styles.alignSelfCenter]}
                                 source={props.icon}
+                                fallbackIcon={props.fallbackIcon}
```

Comment:
Don't we need a size prop here?

Reply:
here avatar has normal height width ( = 40), and we are passing default size prop in Icon is 40 , that'y why no size prop passed here, it will use default value